protocol-models/typescript/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@airbyte/airbyte-protocol",
+  "version": "1.0.0",
+  "description": "",
+  "type": "module",
+  "publishConfig": {
+    "access": "public"
+  },
+  "types": "./dist/index.ts",
+  "scripts": {
+    "build": "node --loader ts-node/esm src/build.ts"
```

Comment:
I think this can be easier just put as `ts-node src/build.ts` (untested, but we usually use the `ts-node` command directly in other places).

Reply:
not with ESM and import now :(

Reply:
cc @bleonard and @pedroslopez - I finally figured out how to run an ESM typescript project without compilation! I think this should also work for `airbytehq/workflow-actions`

Reply:
should still be able to use `ts-node-esm` or `ts-node --esm` according to their [docs](https://github.com/TypeStrong/ts-node#command-line), but functionally won't make any differences ofc.